src/modules/users/types.ts

```diff
@@ -49,6 +49,8 @@ export enum Permission {
 	EDIT_TRANSLATIONS = 'EDIT_TRANSLATIONS',
 	// IDP-based permission:
 	CAN_EDIT_PROFILE_INFO = 'CAN_EDIT_PROFILE_INFO',
+	// Material Requests
+	READ_PERSONAL_APPROVED_MATERIAL_REQUESTS = 'READ_PERSONAL_APPROVED_MATERIAL_REQUESTS',
```

Comment:
why does this permission contain APPROVED?
A name that fits better with the other permissions would be:
```
VIEW_OWN_MATERIAL_REQUESTS
```

Reply:
Adjusted to better represent it's use case